minor: new example for Plots

CeresBarros opened the pull request.

achubaty commented:
> @eliotmcintire, @achubaty I think "R-CMD-check / ubuntu-20.04 (release) (pull_request)" if failing because "Suggests" (and `ggplotify`) aren't being installed. Adding this package to the "Install additional pacakge dependencies" step also didn't seem to work ([94534d0](https://github.com/PredictiveEcology/SpaDES.core/commit/94534d0746675c9cbef738647d41d4d9717b5c02)), so I'm unsure how to proceed

If it's in Suggests but not used conditionally, that will trigger failures for the Ubuntu checks where it does NOT installed suggested packages.

CeresBarros commented:
I believe I've fixed the issue with `ggplotify` -- I had added the `stop` to the example if `ggplotify` wasn't installed. Instead I'm only running the example code if it is installed.
The checks are now failing somewhere else though.
